templates/al2023/runtime/gpu/gpu-ami-util

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+function is-isolated-partition() {
```

Comment:
this is unused, think its because you needed the partition name anyway in the other scripts

Reply:
you're right, I don't think we need this now